src/index.js

```diff
@@ -230,4 +225,51 @@ function strWithLen(len) {
   return new Array(len + 1).join( 'x' );
 }
 
-export { rules };
+function getTagName({ env, tagNameOption}) {
+  if (tagNameOption) {
+    return tagNameOption;
+  } else if (env === 'relay') {
+    return 'Relay.QL';
+  }
+
+  return 'gql';
+}
+
+function getConfig(context) {
+  const {
+    schemaJson, // Schema via JSON object
+    schemaJsonFilepath, // Or Schema via absolute filepath
+    env,
+    tagName: tagNameOption,
+  } = context.options[0];
+
+  return {
+    schemaJson,
+    schemaJsonFilepath,
+    env,
+    tagNameOption
+  }
```

Comment:
This project doesn't have lint rules itself for some reason, but it looks like trailing commas and finishing semicolons are used elsewhere in the original file:
```js
return {
  schemaJson,
  schemaJsonFilePath,
  env,
  tagNameOption,
};
```